src/oop_ext/interface/_interface.py

```diff
@@ -536,70 +536,84 @@ def _IsInterfaceDeclared(class_: Optional[Type], interface: Type[Interface]) ->
     return interface in declared_and_subclasses
 
 
-class Attribute:
-    """"""
-
-    _do_not_check_instance = object()
-
-    def __init__(
-        self, attribute_type: Type, instance: object = _do_not_check_instance
-    ) -> None:
-        """
-        :param type attribute_type:
-            Will check the attribute type in the implementation against this type.
-            Checks if the attribute is a direct instance of attribute_type, or of it implements it.
+if not TYPE_CHECKING:
```

Comment:
The diff is hard to read, but basically indented `Attribute` and `ReadOnlyAttribute` declrations under this `if`, and added the `else:` clause.